2020-02-Spring/2_linked_lists/README.md

```diff
@@ -67,21 +67,60 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Reorder List
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/reorder-list/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a singly linked list
+<a href="https://www.codecogs.com/eqnedit.php?latex=L:&space;L_0&space;\rightarrow&space;L_1&space;\rightarrow&space;...&space;\rightarrow&space;L_{n-1}&space;\rightarrow&space;L_{n}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?L:&space;L_0&space;\rightarrow&space;L_1&space;\rightarrow&space;...&space;\rightarrow&space;L_{n-1}&space;\rightarrow&space;L_{n}" title="L: L_0 \rightarrow L_1 \rightarrow ... \rightarrow L_{n-1} \rightarrow L_{n}" /></a>
+
+Reorder it to: 
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=L:&space;L_0&space;\rightarrow&space;L_n&space;\rightarrow&space;L_1&space;\rightarrow&space;L_{n-1}&space;\rightarrow&space;L_2&space;\rightarrow&space;L_{n-2}&space;\rightarrow&space;..." target="_blank"><img src="https://latex.codecogs.com/gif.latex?L:&space;L_0&space;\rightarrow&space;L_n&space;\rightarrow&space;L_1&space;\rightarrow&space;L_{n-1}&space;\rightarrow&space;L_2&space;\rightarrow&space;L_{n-2}&space;\rightarrow&space;..." title="L: L_0 \rightarrow L_n \rightarrow L_1 \rightarrow L_{n-1} \rightarrow L_2 \rightarrow L_{n-2} \rightarrow ..." /></a>
```

Comment:
These images aren't rendering quite right in the markdown -- not sure why this would be, except that GitHub Markdown has its own limitations.

![rendering](https://user-images.githubusercontent.com/29528527/80654326-cbc57480-8a30-11ea-99d7-a7d31c73c8f1.png)


Also, while it's nice to embed links where we can, downloading the gifs and adding them as images is the best guard against link rot.

Nice LaTeX tho! :D